### PR TITLE
Writing 10m records to PostgreSQL

### LINKED ISSUE #11: write data generation script - 10+ million records for secondary database

### PR DESCRIPTION
**Pull request by allenhsieh**
This commit closes #11.

Was originally going to be a small refactor branch but it ran away from me
1) Refactored seed-Mongo.js to use native mongoDB drivers + seed script now drops any existing `galleries` collections
2) Started on readme (missing installation instructions)
3) Wrote script that generated 10m postgreSQL records 

No documentation for this in readme (since i'm probably going to choose MongoDB) but postgreSQL seed script is `npm run seed-pg`

### CODE REVIEW

File: database/seed-PSQL.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+    });
+  });
+}
+
+const client = new Client({
+  user: process.env.PGUSER,
+  host: process.env.PGHOST,
+  database: process.env.PGDATABASE,
+  password: process.env.PGPASSWORD,
+  port: process.env.PGPORT
```
Comment (cjm771) [inline database/seed-PSQL.js]: need some insight into what .env vars are needed in readme. also tell user to create the database too, i just ran it and it seeded, but went into my public scema database.

Comment (cjm771) [reply database/seed-PSQL.js]:
```
# create database
$> createdb firebnb-galleries
```
Specify in your .env file
```
PGUSER=
PGHOST=
PGDATABASE=firebnb-galleries
PGPASSWORD=
PGPORT=
```

File: database/seed-PSQL.js [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  await client.connect();
+  console.log('Connected to PostgreSQL database');
+  timerStart = Date.now()
+  await client.query('DROP TABLE IF EXISTS galleries')
+  await client.query('CREATE TABLE IF NOT EXISTS galleries (id serial PRIMARY KEY, photos JSON NOT NULL)');
+  try {
+    let rounds = 0;
+    while (rounds < 100) {
+      await csvWriter();
+      await client.query(`COPY galleries(id,photos) FROM '${path.join(__dirname, 'test.csv')}' CSV HEADER`);
```
Comment (cjm771) [inline database/seed-PSQL.js]: some feedback in loop  (wont slow it don)..would be good consider doing 

```
      // in progress loop, clear current line
      process.stdout.clearLine();
      process.stdout.cursorTo(0);
    // write to it
      const MAX_ROUNDS = 100;
      process.stdout.write(`Copying..${Math.floor((rounds / MAX_ROUNDS) * 100)}%..`);
```

**Comment by cjm771** [comment]
great job the comments were just some notes . works great finished in `303 seconds`

**Comment by allenhsieh** [comment]
wonderful notes. thank you @cjm771 !!

**Pull request merged**